clients/src/main/java/org/apache/kafka/clients/consumer/internals/RequestState.java

```diff
@@ -66,6 +67,7 @@ public RequestState(final LogContext logContext,
      * and the backoff is restored to its minimal configuration.
      */
     public void reset() {
+        this.requestInFlight = false;
         this.lastSentMs = -1;
```

Comment:
what's the purpose of `lastSentMs` now? Can we remove it?

Reply:
I've removed `lastSentMs` as it is no longer needed.